src/test/java/com/checkout/payments/RequestApmPaymentsIT.java

```diff
@@ -567,6 +567,7 @@ void shouldMakeCvConnectPayment() {
         checkErrorItem(() -> paymentsClient.requestPayment(paymentRequest), PAYEE_NOT_ONBOARDED);
     }
 
+    @Disabled("Temporarily skipped")
```

Comment:
Why?

Reply:
Sometimes spec changes temporally and the tests fail, and we wait to the next PR to check again. Also these payments depends if our secret key has got enabled this kind of payment.